test.sh

```diff
@@ -181,9 +188,8 @@ if [[ "$RUN" =~ "integration" ]] ; then
     echo "--- Recommend setting \$CERTBOT_PATH to  ---"
     echo "--- client repo with initialized virtualenv  ---"
     echo "------------------------------------------------"
-    run git clone -b v0.8.0 \
-      https://www.github.com/certbot/certbot.git \
-      $CERTBOT_PATH || exit 1
+    run git clone https://www.github.com/certbot/certbot.git $CERTBOT_PATH || exit 1
```

Comment:
I tried the approach @jsha described in https://github.com/letsencrypt/boulder/pull/1920/files/70ec276c0ea6e064c3960ced2ef0b9d1f6d8273a#r69959255 and wasn't able to get it to work. It seems like it requires two operations, a clone and a checkout, unless you can target a specific tag/branch.